lib/GridListImageItem/GridListImageItem.css

```diff
@@ -14,6 +14,10 @@
 	height: 100%;
 	max-height: 100%;
 }
+.enyo-gridlist-imageitem .enyo-gridlist-imageitem-image {
```

Comment:
This is fine, functionally, but it is a bit odd semantically that we only add this class under the specific case where we are not using a background-image and have specified a placeholder image; the name of the class itself seems too generic in this case.